src/AuthenticationService.php

```diff
@@ -85,7 +85,8 @@ class AuthenticationService implements AuthenticationServiceInterface
     protected $_defaultConfig = [
         'authenticators' => [],
         'identifiers' => [],
-        'identityClass' => Identity::class
+        'identityClass' => Identity::class,
```

Comment:
I don't think this should be here. It should be IdentityInterfacen::class and when we use the interface there is no need to make it configurable. We want to enforce the interface as a minimum common API of identities. There should be no way to bypass that.